src/pages/components/form/accessibility.mdx

```diff
@@ -17,102 +17,75 @@ import {
-        </StructuredListRow>
-      </StructuredListHead>
-      <StructuredListBody>
-        <StructuredListRow>
-          <StructuredListCell>
-            - macOS Mojave version 10.14.2 with VoiceOver
-            <br />
-            - Chrome version 77.0.3865.90
-            <br />
-            - Dynamic Assessment Plugin (DAP) version 1.8.0.0 - IBM
-            Accessibility WCAG 2.1 Sept. 2019 Ruleset
-            <br />- Carbon React version 7.7.1
-          </StructuredListCell>
-          <StructuredListCell>
-            <strong>DAP test:</strong>
-            <br />- No violations
-          </StructuredListCell>
-        </StructuredListRow>
-      </StructuredListBody>
-    </StructuredListWrapper>
-  </Column>
+<Column colLg={8}>
+
+![example of information icon taking separate focus in a text input](images/form-accessibility-1.png)
+
+<Caption>
+  The toggletip pattern is used to make form help text keyboard accessible.
+</Caption>
+
+</Column>
 </Row>
+
+### Error handling
+
+Carbon incorporates accessible inline error and warning messages into many components, such as text inputs. Error states are also conveyed programmatically to assistive technologies.
+
+
+## Design recommendations
+
+Design annotations and considerations are needed for the following situations.
+
+### Identify requirements at the start of the form
+
+Traditionally, a legend at the start of a form identifies the symbol (often an asterisk) used for required fields, and the symbol is repeated as part of the label for each appropriate field. This is still considered the most accessible implementation. However, as discussed in [Optional versus required fields](/components/form/usage/#optional-versus-required-fields), Carbon allows for either the required or optional fields to be the inputs identified. Especially where only optional fields are indicated, an instruction should precede a form, providing the context for whether required or optional fields are indicated. The traditional phrasing is “All fields are required unless marked as optional” (or the reverse). See the Equal Access Toolkit topic [Required fields](https://www.ibm.com/able/toolkit/design/visual/#required-fields) for more information.
+
+By convention, simple username/password login forms do not need such an instruction (or even to be marked as required), since the context is clear to users.
+
+### Be familiar with the accessibility of common form components
+
+The following topics each have their own accessibility considerations, which improve the overall form experience:
+
+- Checkbox
+-	Date picker
+-	Dropdown
+-	File uploader
+-	Loading
+-	Modal
+-	Notification
+-	Number input
+-	Radio button
+-	Select
+-	Slider
+-	Text input
+-	Toggle tip
+
+## Development considerations
+
+Keep these considerations in mind if you are modifying Carbon patterns or creating a custom form or input component:
+- a form must be wrapped in a `<form>` element.
```

Comment:
These bullet points need to all start with a capital letter and none of them should have a period on the end (since they're statements, that's how we format lists elsewhere)

```suggestion
- A form must be wrapped in a `<form>` element
```

Reply:
I have retained the terminal punctuation, but added sentence case _and_ turned the colon in the preamble into a period, to reduce any confusion. Confirmed list style with Jan.